Xamarin.UserConsent/UI/RecordingConsentView.xaml

```diff
@@ -0,0 +1,83 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentView xmlns="http://xamarin.com/schemas/2014/forms" 
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:themes="clr-namespace:Xamarin.UserConsent.Resources;assembly=Xamarin.UserConsent"
+             mc:Ignorable="d"
+             x:Name="RecordingConsent"
+             x:Class="Xamarin.UserConsent.UI.RecordingConsentView">
+    <ContentView.Content>
+        <StackLayout
+          BindingContext="{x:Reference RecordingConsent}"
+          BackgroundColor="White"
+          Padding="0, 20, 0, 0">
+            <Label
+              Margin="0, 15"
+              HorizontalOptions="Center"
+              Text="{Binding Header}"
+              FontSize="{x:Static themes:FontSizes.MainHeadingLabelFontSize}" />
```

Comment:
For now, static resources are used to allow others to work with this view.  It could be changed to use defined CSS.

Reply:
Please add a ticket so that it doesn't get lost.